lib/index.d.ts

```diff
@@ -109,17 +109,18 @@ declare namespace Component {
         useShadowDom?: boolean;
     }
 
-    type AttrType = 'string' | 'number' | 'boolean' | 'json';
-
     interface AttrSchema {
-        /** Type of the attribute */
-        type: AttrType;
+        /** Type of the attribute. One of 'string' | 'number' | 'boolean' | 'json' */
+        type: string;
```

Comment:
whoa. can't do enums??

Reply:
yeah tsc 3 is being a PITA: `Type 'string' is not assignable to type '"string" | "number" | "boolean" | "json" | AttrSchema'.`

Not the end of the world because we do a static check anyway: https://github.com/mixpanel/panel/blob/master/lib/component.js/#L452